exercise-06-package-sync/build.gradle.kts

```diff
@@ -34,6 +35,17 @@ dependencies {
 }
 
 aem {
+    fileTransfer {
+        smb {
```

Comment:
Idea was to use credentials method which sets for all protocols.. Then override single protocol / sftp as example